README.md

```diff
@@ -52,17 +52,18 @@ You will need to make a PR after committing the changes to this file.
 
 ## Using `mathlib4` as a dependency
 
-If you want to start a project that uses `mathlib4` as a dependency, you can run:
-```
-lake new MyProject math
-```
+To start a new project that uses mathlib4 as a dependency:
+
+1. Open a folder that contains no file named `lean-toolchain`.
+2. Update your toolchain to one that is not less recent than the toolchain specified in the `lean-toolchain` file of this repository's master branch.
```

Comment:
It's better to suggest the command itself here: `elan default leanprover/lean4:nightly-2023-02-04`

Reply:
This was about a specific toolchain right? This will have to be updated every time we change the toolchain for mathlib4 itself. It might cause people to Ctrl C + Ctrl V the command and face a wall of errors.

Reply:
I added some less toolchain specific code